NAMESPACE

```diff
@@ -1 +1,27 @@
 # Generated by roxygen2: do not edit by hand
+
+export(Create_continuous_age_variable)
+export(Create_discrete_age_variable)
+export(Create_individuals)
+export(Create_states)
+export(E2_IMild)
+export(Get_parameters_for_sirstochastic)
+export(Individual_I_to_R_2)
+export(Individual_I_to_R_I2_to_D)
+export(Individual_R_to_S)
+export(Individual_R_to_S_2)
+export(Individual_S_to_I)
+export(Individual_S_to_I_2)
+export(Individual_S_to_I_and_I2)
+export(Infections)
+export(Probabilities_of_states)
+export(Render_state_sizes)
+export(Render_state_sizes2)
+export(Render_state_sizes3)
+export(Render_state_sizes4)
+export(Render_state_sizes_2)
+export(SEIRexplicitparameters)
+export(Validated_state_update)
```

Comment:
Do all of these have to be exported?

What do clients need except for a `get_parameters` and `run_hypatia` type?